### PR TITLE
[Windows] Add new test case 'power_operation_scripts'

### PR DESCRIPTION
**Pull request by ZouYuhua**
replace merge request #648

### CODE REVIEW

**Comment by ZouYuhua** [comment]
Test passed for windows 2025 and window vNext.

**Review by Tomorrow9 — CHANGES_REQUESTED** [review]

File: windows/power_operation_scripts/set_power_script_custom.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check customized power operation script
+#
+- name: "Set fact of customized script file"
+  ansible.builtin.set_fact:
+    script_file_path: "C:\\ProgramData\\VMware\\VMware Tools\\{{ power_cmd_op }}-vm-custom.bat"
+    script_tag_path: "C:\\ProgramData\\VMware\\VMware Tools\\{{ power_cmd_op }}.tag"
+    script_file_in_conf_file: "C:\\ProgramData\\VMware\\VMware Tools\\{{ power_cmd_op }}-vm-custom.bat"
```
[the remark this answers is not in the document]
Comment (ZouYuhua) [reply windows/power_operation_scripts/set_power_script_custom.yml:11]: Yes. It's expected.

[the remark this answers is not in the document]
Comment (Tomorrow9) [reply windows/power_operation_scripts/set_power_script_custom.yml:11]: Then you can write it in a correct way.

**Review by Tomorrow9 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**